opentracing/span.py

```diff
@@ -215,15 +215,16 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         and added as a tag. :attr:`~operation.ext.tags.ERROR` will also be set
         to `True`.
         """
-        self._on_error(exc_type, exc_val, exc_tb)
+        self._on_error(self, exc_type, exc_val, exc_tb)
```

Comment:
should this be `Span._on_error`? I'd expect some linter to complain.

Reply:
I don't think so, but I'll change it if you consider it necessary.

Reply:
It has been changed to `Span._on_error`.